cards/src/main/resources/cards/custom/group2/set_633/hero_grandmaster_stormstout.json

```diff
@@ -14,7 +14,7 @@
         "minion_student_of_the_crane",
         "minion_student_of_the_ox",
         "minion_student_of_the_serpent",
-        "minion_student_of_the_tiger"
+        "minion_lungrath_hunter"
```

Comment:
ID changes need a corresponding migration in the `Spellsource.java` file, that will resemble something of the form

```
				.add(new MigrationRequest()
						.withVersion(31)
						.withUp(thisVertx -> {
							changeCardId(""minion_student_of_the_tiger", "minion_lungrath_hunter");
						}))
				.migrateTo(31, then2 ->
						then.handle(then2.succeeded() ? Future.succeededFuture() : Future.failedFuture(then2.cause())));
```

Observe the migration number is incremented, and you will add additional `changeCardId` calls